README.md

```diff
@@ -140,6 +140,7 @@
   * [**C**: *Tutorial - Write a Shell in C*](https://brennan.io/2015/01/16/write-a-shell-in-c/)
   * [**C**: *Let's build a shell!*](https://github.com/kamalmarhubi/shell-workshop)
   * [**Ruby**: *A Unix Shell in Ruby*](https://www.jstorimer.com/blogs/workingwithcode/7766107-a-unix-shell-in-ruby)
+  * [**C**: *Writing a UNIX Shell - Part I*](https://indradhanush.github.io/blog/writing-a-unix-shell-part-1/)
```

Comment:
Great work! Can you please move the row above the Ruby example so that the links are grouped by the programming language?

Reply:
Fixed!